core/domain/exp_domain_test.py

```diff
@@ -344,6 +344,40 @@ def test_validation(self):
                 utils.ValidationError, 'destination ABC is not a valid'):
             exploration.validate()
 
+        # Ensure an answer group with two fuzzy rules is invalid
+        init_state = exploration.states[exploration.init_state_name]
+        default_outcome = init_state.interaction.default_outcome
+        default_outcome.dest = exploration.init_state_name
```

Comment:
Is there something to restore at this point? The [before version](https://github.com/oppia/oppia/blob/develop/core/domain/exp_domain_test.py#L347) doesn't seem to do anything.